WordPress.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,6 +1,87 @@
 {
   "object": {
     "pins": [
+      {
+        "package": "AutomatticAbout",
```

Comment:
I do. I think we're seeing these changes because they were deleted in https://github.com/wordpress-mobile/WordPress-iOS/commit/bac6644ad006d6bac410dbe4431be70ea930f77c

Reply:
If we revert the changes made in https://github.com/wordpress-mobile/WordPress-iOS/commit/bac6644ad006d6bac410dbe4431be70ea930f77c, we won't see package.resolved changes going forward, right?

Reply:
Hmm, OK let me revert that. I'll merge it with this PR.